cli/flox/doc/flox-activate.md

```diff
@@ -80,10 +80,26 @@ hook was defined.
 
 # ENVIRONMENT VARIABLES
 
-## `$_FLOX_ACTIVE_ENVIRONMENTS`
-A JSON array containing one object per active environment.
-This is currently an implementation detail
-and its contents are subject to change.
+`$_FLOX_ACTIVE_ENVIRONMENTS`
+:   A JSON array containing one object per active environment.
+    This is currently an implementation detail
+    and its contents are subject to change.
+
+`$FLOX_PROMPT_ENVIRONMENTS`
+:   A JSON array containing one object per active environment.
```

Comment:
The description needs to be changed too lol

Reply:
Fixed